feat(landing-page): redesigned footer with improved appearance, functionality, and countdown section

NerioNitharshan opened the pull request.

NerioNitharshan committed:
Re Design

NerioNitharshan committed:
feat(landing-page): redesigned hero and footer for improved appearance and functionality

NerioNitharshan committed:
feat(landing-page): redesigned footer with improved appearance, functionality, and countdown section

NerioNitharshan committed:
Error fix

BirnadinErick commented on package-lock.json:
we use pnpm, please delete this

NerioNitharshan replied:
done

BirnadinErick commented on api/v1/controllers/Projects.php:
somehow this push is different from what we've seen on the meeting

BirnadinErick commented on tailwind.config.cjs:
please re-confirm change in these colors.

NerioNitharshan replied:
no change till now

NerioNitharshan commented:
Okay

On Monday, 11 November 2024, Birnadin Erick ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> On package-lock.json
> <https://github.com/BirnadinErick/spc-175/pull/146#discussion_r1836736476>
> :
>
> we use pnpm, please delete this
> ------------------------------
>
> On api/v1/controllers/Projects.php
> <https://github.com/BirnadinErick/spc-175/pull/146#discussion_r1836737117>
> :
>
> somehow this push is different from what we've seen on the meeting
> ------------------------------
>
> On tailwind.config.cjs
> <https://github.com/BirnadinErick/spc-175/pull/146#discussion_r1836740983>
> :
>
> please re-confirm change in these colors.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BirnadinErick/spc-175/pull/146#pullrequestreview-2427397527>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANTHUCJGKRV7BPBZYFX7E2D2AC4H7AVCNFSM6AAAAABP3JQUASVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDIMRXGM4TONJSG4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

NerioNitharshan committed:
Update MobileNavigationMenu.astro

NerioNitharshan committed:
Delete package-lock.json

NerioNitharshan committed:
deleted old footer component

NerioNitharshan committed:
error fix

NerioNitharshan committed:
fix

NerioNitharshan committed:
.

NerioNitharshan committed:
fix

NerioNitharshan committed:
fix(package.json): update dependencies
fix(package-lock.json): update dependencies
fix(pnpm-lock.yaml): update dependencies
fix(postcss.config.js): update postcss-preset-env
